src/app/screens/HomeScreen/dashboard.hooks.ts

```diff
@@ -5,7 +5,7 @@ import {getTimesheetCalendar} from '../../services/home';
 export const useHomeCalendar = (month: string, year: number) => {
   const {data, isLoading} = useQuery({
     queryKey: ['home_calendar_data', month, year],
-    queryFn: async () => getTimesheetCalendar(month, year),
+    queryFn: () => getTimesheetCalendar(month, year),
     onError: error => {
       console.log(error);
     },
```

Comment:
instead of a console add an error toast here and show the backend error message if present else show `Something went wrong while fetching calendar data`